docs/guidelines/platforms/android/guideline_percievable_android.md

```diff
@@ -23,23 +23,60 @@ If an element exists on the UI only for decorative purposes it is recommended th
 
 :white_check_mark: **Success criteria**
 
-Suppose the app you are building includes media content such as video clips or audio recordings. In that case, it is suggested to provide an alternative for users with different types of accessibility needs for them to understand the material. The suggested practice would be to:
+##### Using ExoPlayer with captions
 
-- Include controls that allow users to pause and stop media, change volume and toggle subtitles (captions)
+One way to enhance accessibility is by adding captions, either open or closed. Open captions are permanently visible as they are integrated into the video, while closed captions require a media format with a player that supports them. Most modern video players support captions, and `ExoPlayer` provides built-in support for displaying them automatically.
 
-- If a video presents information that is vital to completing workflow, provide the same content in an alternative format, such as a transcript
+To view captions in media that supports them, users must enable the _Show captions_ option. This accessibility feature can be found under **Settings** > **Accessibility** > **Caption preferences**. On this screen, users can also adjust caption size and style; however, these preferences may not work with media apps that don’t support Caption Preferences.
 
-:no_entry_sign: **Failure criteria**
+##### Custom solution
+
+If a different player is used instead of `ExoPlayer`, the status of closed captions, along with the font scale and caption style, can be checked programmatically using the `CaptioningManager`. When the _Show captions_ option is enabled, captions should be turned on in the selected player.
+
+```kotlin
+import android.content.Context
+import android.view.accessibility.CaptioningManager
+import android.view.accessibility.CaptioningManager.CaptionStyle
+
+class CaptionUtils(context: Context) {
+    
+   private val captioningManager: CaptioningManager =
+       context.getSystemService(Context.CAPTIONING_SERVICE) as CaptioningManager
+    
+   fun areCaptionsEnabled(): Boolean = captioningManager.isEnabled
+    
+   fun getCaptionFontScale(): Float = captioningManager.fontScale
+    
+   fun getCustomCaptionStyle(): CaptionStyle = captioningManager.userStyle
+}
+```
+
+### Captions support for live media (WCAG 1.2.4 - Level AA)
+
+This guideline covers [1.2.4 Captions (Live) - Level AA](https://www.w3.org/WAI/WCAG22/quickref/#captions-live) of the WCAG standard.
+
+In some cases, live media is used in the application. To ensure accessibility for all users, providing captions for live content is important.
+
+:white_check_mark: **Success criteria**
+
+For all live media in the application, captions should be available through the user interface. This can be achieved in two ways:
```

Comment:
I would maybe add the exemption from the docs here, just to be a bit more cautious about what the application should/shouldn't handle. 
Something like this:

All live media in the application should offer captions through the user interface, available as either open or closed captions. For two-way multimedia calls between individuals, accessibility requirements are the responsibility of content providers, not the application.

**Open captions**: ...